readthedocs/rtd_tests/fixtures/spec/v2/schema.yml

```diff
@@ -77,6 +77,22 @@ python-install:
   # Default: []
   extra_requirements: list(str(), required=False)
 
+python-install-pipfile:
+  # The path to the project to be installed
+  pipfile: path()
```

Comment:
I'm tempted to left this as `path` and include `pipfile` as option for `method

https://github.com/rtfd/readthedocs.org/blob/df85fefc5f59c7a00bcc1ffe23965efe75e8c895/readthedocs/rtd_tests/fixtures/spec/v2/schema.yml#L67-L73

The options `dev`, `ignore_pipfile` and `skip_lock` would stay.

Reply:
I'm not sure that would be convenient. We are using `path` only when we need to install a project that it's on the same repository that the docs we are building.

Also, we have the `requirements` options that _to me_ looks similar that `pipfile`. So, I think it's more expressive to say `pipfile` than `path: docs/Pipfile` + `method: pipfile`.

Reply:
i lean towards `pipfile: path()` as well. Pipfile support operates like requirements installation in that it skips setup.py and pip installation of the package itself, it just installs dependencies. So it feels a little disjointed to lump in with the same schema for package installation.

Also, definitely agree on `pipfile: path()` being more expressive as well